src/components/User/Limits/components/Limits.Manager.vue

```diff
@@ -319,13 +310,39 @@
         }
         this.isPending = false
       },
+      async removeLimits (limits) {
+        this.isPending = true
 
+        if (limits.id === 0) {
+          this.isPending = false
+          return false
+        }
+        if (!await confirmAction({ title: 'Delete the limits?' })) return
+        try {
+          const operation = Sdk.base.ManageLimitsBuilder.removeLimits({
+            id: (limits.id).toString()
+          })
+          await Sdk.horizon.transactions.submitOperations(operation)
```

Comment:
Pls replace with the new analogue from `ApiCallerFactory`. `Sdk.horizon.transactions` is going to be removed soon